scripts/check-version.js

```diff
@@ -26,22 +26,26 @@ const spawnSync = (...args) => {
 }
 
 if (!fs.existsSync(path.join('node_modules', 'semver'))) {
-    spawnSync(npm, [
-        'install',
-        `semver@${pkg.devDependencies['semver']}`,
-        '--loglevel=silent',
-        '--no-save',
-        '--no-package-lock',
-        '--ignore-scripts',
-        '--no-audit'
-    ])
+    spawnSync(
+        npm,
+        [
+            'install',
+            `semver@${pkg.devDependencies['semver']}`,
+            '--loglevel=silent',
+            '--no-save',
+            '--no-package-lock',
+            '--ignore-scripts',
+            '--no-audit'
+        ],
+        {shell: true}
+    )
```

Comment:
What is this `{shell: true}` change about. And can you mention this in the description at a minimum? maybe even another pr so it's on the record and not hidden away in this version bump.

Reply:
That was merged in during the v3.5.0 release. So i'm not going to create a separate PR.

I updated the PR description